install: fix impossibility of reinstalling master (#888)
tarantool/tt: CHANGELOG.md

- Sorted by name order of columns for `table` and `ttable` formats.
- `tt switch tt`: does not work with `x.x.x` version format.

> - Added ability to install latest version of `master` tarantool.

Reviewer: We already have `Added` a couple of lines at top.

```suggestion
- Ability to update a latest version of `master` tarantool with `tt install`.
```

Author: Changed it.
